src/org/jitsi/impl/neomedia/transform/srtp/SRTPCryptoContext.java

```diff
@@ -507,8 +505,8 @@ public void processPacketAESCM(RawPacket pkt)
 
         ivStore[14] = ivStore[15] = 0;
 
-        int payloadOffset = pkt.getHeaderLength();
-        int payloadLength = pkt.getPayloadLength();
+        int payloadOffset = RawPacket.getHeaderLength(pkt.getBuffer(), pkt.getOffset(), pkt.getLength());
+        int payloadLength = pkt.getLength() - payloadOffset;
```

Comment:
here we could use one of the overloads of `RawPacket.getPayloadLength` if we want to

Reply:
I made this simpler, but I don't want to use `getPayloadLength` because it recalculates the header length (same below).